ui/packages/app/index.html

```diff
@@ -44,7 +44,7 @@
 			crossorigin="anonymous"
 		/>
 		<link
-			href="https://fonts.googleapis.com/css?family=Source Sans Pro"
+			href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap"
```

Comment:
@pngwn font-semibold was never imported? on purpose?

Reply:
Most text is rendered with Source Sans Pro, IBM Plex Mono is only used for the mono font. Also we use [semibold weight](https://github.com/search?q=repo%3Agradio-app%2Fgradio%20font-semibold&type=code) in the codebase.

Reply:
yeah looks like our semibolds weren't rendering, good catch